Source/SwiftLintFramework/Rules/FileHeaderRule.swift

```diff
@@ -9,7 +9,9 @@ public struct FileHeaderRule: ConfigurationProviderRule, OptInRule {
     public static let description = RuleDescription(
         identifier: "file_header",
         name: "File Header",
-        description: "Header comments should be consistent with project patterns.",
+        description: "Header comments should be consistent with project patterns. " +
```

Comment:
Can you please provide the `reason` with the old `description` when creating `StyleViolation`s?

This way oss-check won't report diffs and it'll also be easier for people to read the warnings.

Reply:
I was concerned about all those diffs reported by ops-check.
Thanks for teaching me how to fix!

Btw: Had to do small reformat to avoid function_body_length error due to the two extra "reason" lines added in ```validate(file:)```.